app/src/main/java/com/oscarg798/pokedesk/MainActivity.kt

```diff
@@ -37,6 +44,18 @@ class MainActivity : ComponentActivity() {
     }
 }
 
+private fun animationForSplashScreen(splashScreenView: SplashScreenViewProvider) {
+    val alpha = ObjectAnimator.ofFloat(
+        splashScreenView.view,
+        View.ALPHA,
+        1f,
+        0f
+    )
+    alpha.duration = 3000L
```

Comment:
I really love it, the only comment I have is `3000L` seems too much to me, I would use `500L`

Reply:
You're right!